source/util/move_to_front.h

```diff
@@ -103,172 +110,172 @@ class MoveToFront {
+    return nodes_.at(node).size;
   }
 
-  // Returns immutable height of the node referenced by |handle|.
-  inline uint16_t HeightOf(uint16_t handle) const {
-    return nodes_.at(handle).height;
+  // Returns height of |node|.
+  uint32_t HeightOf(uint32_t node) const {
+    return nodes_.at(node).height;
   }
 
-  // Returns mutable id of the node referenced by |handle|.
-  inline uint32_t& MutableIdOf(uint16_t handle) {
-    assert(handle);
-    return nodes_.at(handle).id;
+  // Returns mutable reference to id of |node|.
+  uint32_t& MutableIdOf(uint32_t node) {
+    assert(node);
+    return nodes_.at(node).id;
   }
 
-  // Returns mutable handle to left of the node referenced by |handle|.
-  inline uint16_t& MutableLeftOf(uint16_t handle) {
-    assert(handle);
-    return nodes_.at(handle).left;
+  // Returns mutable reference to handle of left child of |node|.
+  uint32_t& MutableLeftOf(uint32_t node) {
+    assert(node);
+    return nodes_.at(node).left;
   }
 
-  // Returns mutable handle to right of the node referenced by |handle|.
-  inline uint16_t& MutableRightOf(uint16_t handle) {
-    assert(handle);
-    return nodes_.at(handle).right;
+  // Returns mutable reference to handle of right child of |node|.
+  uint32_t& MutableRightOf(uint32_t node) {
+    assert(node);
+    return nodes_.at(node).right;
   }
 
-  // Returns mutable handle to parent of the node referenced by |handle|.
-  inline uint16_t& MutableParentOf(uint16_t handle) {
-    assert(handle);
-    return nodes_.at(handle).parent;
+  // Returns mutable reference to handle of parent of |node|.
+  uint32_t& MutableParentOf(uint32_t node) {
+    assert(node);
+    return nodes_.at(node).parent;
   }
 
-  // Returns mutable timestamp of the node referenced by |handle|.
-  inline uint32_t& MutableTimestampOf(uint16_t handle) {
-    assert(handle);
-    return nodes_.at(handle).timestamp;
+  // Returns mutable reference to timestamp of |node|.
+  uint32_t& MutableTimestampOf(uint32_t node) {
+    assert(node);
+    return nodes_.at(node).timestamp;
   }
 
-  // Returns mutable size of the node referenced by |handle|.
-  inline uint16_t& MutableSizeOf(uint16_t handle) {
-    assert(handle);
-    return nodes_.at(handle).size;
+  // Returns mutable reference to size of |node|.
+  uint32_t& MutableSizeOf(uint32_t node) {
+    assert(node);
+    return nodes_.at(node).size;
   }
 
-  // Returns mutable height of the node referenced by |handle|.
-  inline uint8_t& MutableHeightOf(uint16_t handle) {
-    assert(handle);
-    return nodes_.at(handle).height;
+  // Returns mutable reference to height of |node|.
+  uint32_t& MutableHeightOf(uint32_t node) {
+    assert(node);
+    return nodes_.at(node).height;
   }
 
-  // Returns true iff |handle| is left child of its parent.
-  inline bool IsLeftChild(uint16_t handle) const {
-    assert(handle);
-    return LeftOf(ParentOf(handle)) == handle;
+  // Returns true iff |node| is left child of its parent.
```

Comment:
This is intentional, iff in this context means if and only if.
https://en.wikipedia.org/wiki/If_and_only_if